Tasks/PublishBuildArtifactsV1/Invoke-Robocopy.ps1

```diff
@@ -85,7 +86,7 @@ if ($ParallelCount -gt 1) {
     }
 }
 else {
-    & robocopy.exe /E /COPY:DA /NP /R:3 $Source $Target $File 2>&1 |
+    & robocopy.exe $CopySubFoldersOption /COPY:DA /NP /R:3 $Source $Target $File 2>&1 |
```

Comment:
Could you please add some unit tests to cover it?

Reply:
We don't have tests that covers this ps1 file. And I'm not sure how this function can be tested. Mostly this script contains invocation of robocopy.exe and I an not sure how I can mock it

Reply:
Ok, this sounds like a more wide task we could pick up later